Event toggling snippet

AlexJerabek opened the pull request:
This sample shows how to turn on and off events.

My intention is to make this part of a larger sample once calculation toggling goes online. Hence why there's more code and data here than absolutely necessary.

ElizabethSamuel-MSFT commented on samples/excel/89-preview-apis/events-disable-events.yaml:
```diff
@@ -0,0 +1,166 @@
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to turn on and off events.</p>
```
maybe "turn events on and off" instead

ElizabethSamuel-MSFT commented on samples/excel/89-preview-apis/events-disable-events.yaml:
```diff
@@ -0,0 +1,166 @@
+            });
+        }
+
+        async function setup() {
```
I recommend that you have setup create a new sheet with some appropriate data. Then you don't have to worry about interfering with the user's sheets and the other methods can look for that sheet.

AlexJerabek commented:
@ElizabethSamuel-MSFT Thanks for the feedback. Updated accordingly.

I also changed what the event handler does to be more obvious when it's on or off.

ElizabethSamuel-MSFT commented on samples/excel/89-preview-apis/events-disable-events.yaml:
```diff
@@ -0,0 +1,177 @@
+                const cell = sheet.getRange("P21");
+                 
+                // get the grand total of the whole sum range
+                // note that we are using having this program perform the sum instead of giving the Excel cell a function,
```
Should this start with: // note that we are having
